bindings/python/IK/src/JointTrackingTask.cpp

```diff
@@ -26,22 +26,22 @@ void CreateJointTrackingTask(pybind11::module& module)
     using namespace BipedalLocomotion::IK;
     using namespace BipedalLocomotion::System;
 
-    py::class_<JointTrackingTask, std::shared_ptr<JointTrackingTask>, LinearTask>(module, "JointTrackingTask")
+    py::class_<JointTrackingTask, std::shared_ptr<JointTrackingTask>, LinearTask>(module,
+                                                                                  "JointTrackingTas"
+                                                                                  "k")
```

Comment:
same

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/303/commits/f1fe0a34d5d2905e16d36e787af0ea54c6faf7a8